tests/module-entry-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var assert = require('assert');
+var ModuleEntry = require('../lib/module-entry');
+
+describe.skip('ModuleEntry', function() {
```

Comment:
@dcombslinkedin can you implement these tests

Reply:
done (at least to the point I can close this).